Makefile

```diff
@@ -19,3 +19,28 @@ fmt:
 
 check: 
 		docker compose -f docker/docker-compose.yaml run websocket-api bash -c "cd /app && cargo clippy --all  -- --deny warnings && cargo fmt --check"
+
+clean:
+		docker compose -f docker/docker-compose.yaml down --remove-orphans
```

Comment:
this is pretty good, I think that 

```
 docker compose -f docker/docker-compose.yaml down \
       --remove-orphans  \
       --volumes         \
       --rmi all
       ```
will delete all volumes and images in the compose file:

```
      --remove-orphans   Remove containers for services not defined in the Compose file
      --rmi string       Remove images used by services. "local" remove only images that
                         don't have a custom tag ("local"|"all")
  -t, --timeout int      Specify a shutdown timeout in seconds
  -v, --volumes          Remove named volumes declared in the "volumes" section of the
                         Compose file and anonymous volumes attached to containers
```